docs/REST API/rest-api.md

```diff
@@ -6600,6 +6601,63 @@ The timeout period is configurable via environment variable on the Fleet server
 }
 ```
 
+
+### Run custom live query
+
+Runs a custom live query against the specified hosts and responds with the results.
+
+If some targeted hosts haven't responded, the live query will stop and return all collected results after 25 seconds (or whatever time period is configured via environment variable, e.g. `FLEET_LIVE_QUERY_REST_PERIOD=90s`).
+
+
+`POST /api/v1/fleet/queries/custom`
```

Comment:
Or possibly:
```suggestion
`POST /api/v1/fleet/queries/custom/run`
```
(reminder: `/queries/run` is already in use by the websocket endpoint, so that's off the table)